plugins/flytekit-pydantic/flytekitplugins/pydantic/flytepath_creation.py

```diff
@@ -14,6 +14,16 @@
 FlytePath = TypeVar("FlytePath", flyte_file.FlyteFile, flyte_directory_types.FlyteDirectory)
 
 
+def upload_to_s3(flytepath: FlytePath, ctx: Optional[context_manager.FlyteContext] = None) -> None:
```

Comment:
nit: could we remove `ctx: Optional[context_manager.FlyteContext] = None`? and always create `ctx` in this function.